svm/src/transaction_processor.rs

```diff
@@ -648,7 +641,7 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
                 executed_units,
                 accounts_data_len_delta,
             },
-            programs_modified_by_tx: Box::new(programs_modified_by_tx),
+            programs_modified_by_tx: program_cache_for_tx_batch.get_modified_entries().clone(), // Yuck, but temporary.
```

Comment:
@Lichtso I marked this a "temporary" because, if you like the hoisting of global program cache updating, we can actually remove this field from the transaction result altogether. Lmk.